picard/ui/itemviews.py

```diff
@@ -976,7 +976,7 @@ def update(self, update_tracks=True):
                     item.update(update_album=False)
         if album.errors:
             self.setIcon(MainPanel.TITLE_COLUMN, AlbumItem.icon_error)
-            self.setToolTip(MainPanel.TITLE_COLUMN, _("Error"))
+            self.setToolTip(MainPanel.TITLE_COLUMN, _("Processing error(s): See the Errors tab in the Album Info popup"))
```

Comment:
If we want to stick with what the online docs use for all dialogs:

```suggestion
            self.setToolTip(MainPanel.TITLE_COLUMN, _("Processing error(s): See the Errors tab in the Album Info dialog"))
```